script/Ciel/Day 10/10_01_CIEL10_1_1.txt

```diff
@@ -179,8 +179,8 @@ It only lasts a moment.
 [sha:b5ea8f5b9c3bf88ff3e52c54bf891633ba35c94a]{
 -- Page 47, Offset 27192.
 -- 一瞬だけ、今朝の頭痛とは違う、沈むような眩暈がした。
-A wave of vertigo washes over me and is gone.
- It didn't make my dizzy like my headache did this morning,
+A wave of vertigo washes over me and fades.
```

Comment:
Prefer the `is gone` here - since this is 一瞬だけ / 'only for an instant', I prefer the wave washing over -> passing immediately imagery to 'fades', which implies a more gradual process imo. Good catch on the `me/my` though